apps/docs/src/guide/contracts/variable-outputs.md

```diff
@@ -2,21 +2,21 @@
 
 # Variable Outputs
 
-You may need to send funds to the transaction output in certain scenarios. Sway provides a method called `transfer_to_address(coins, asset_id, recipient)` that we can use for this purpose, which allows you to transfer a specific number of coins for a given asset to a recipient address.
+You may need to send funds to the transaction output in certain scenarios. Sway provides a method called `transfer_to_address(recipient, asset_id, amount)` that we can use for this purpose, which allows you to transfer a specific number of coins for a given asset to a recipient address.
```

Comment:
Was this changed recently in Sway's std lib and we failed to pick it up?

Reply:
Nice catch @Dhaiwat10. I've made this change by mistake because I made a similar change to our Token contracts. 

I am really happy that you find this 😅

Reply:
It turns out that it was right the first time 😓 . 

Therefore, I reverted the [change](https://github.com/FuelLabs/fuels-ts/pull/1787/commits/40520bcb25ad1d59ef5c6931eb8c84d5a6e90854)

Since we were using a hardcoded code snippet, this was wrong for quite some time.

 I am fixing on https://github.com/FuelLabs/fuels-ts/pull/1801